packages/@angular/cli/blueprints/ng/files/package.json

```diff
@@ -12,11 +12,13 @@
   },
   "private": true,
   "dependencies": {
+    "@angular/animations": "^4.0.0",
```

Comment:
@dave11mj About #5786; importing Animations into the NgModule has a lot of impact on the rendering and it can negatively affect performance if you're not using animations. As such we just add it to the package.json to facilitate you importing it, if you need to, but we don't force you to remove it if you don't. It will not be included in the build if you don't need it.